test/bundler/transpiler/transpiler.test.js

```diff
@@ -3235,6 +3235,18 @@ console.log(foo, array);
       expect(exports[1]).toBe("default");
       expect(exports).toHaveLength(3);
     });
+
+    it("#17961", async () => {
```

Comment:
Can you add a variation of this test that runs for `"target": "browser"` and ``"target": "bun"`? IIRC, we use a different output encoding

Reply:
It doesn't work properly with `bun` as a target will take a look

the output:
```ts
let list = ["\u2022", "-", "\u25E6", "\u25AA", "\u25AB"];
```

Reply:
@Jarred-Sumner fixed

Reply:
It is expected behaviour for --target=bun to output as ascii only

Reply:
@pfgithub should be good to go